grammars/ruby.cson

```diff
@@ -1940,6 +1952,10 @@
     'name': 'punctuation.separator.key-value'
   }
   {
+    'match': '->'
+    'name': 'keyword.operator.lambda.ruby'
```

Comment:
> Perhaps `lambda` is miscategorized, since it is part of core and not support?

`support.function` is for functions provided by the framework/library (e.g. `puts`)
See [TextMate naming conventions for details](http://manual.macromates.com/en/language_grammars#naming_conventions)

> How about `storage.type.function.arrow` (as you suggested) and change `lambda` to `storage.type.function`

`storage.type` is correct scopes for them. However I almost forget that changing root scopes will change syntax highlining:

<!-- -->
> ![atom_dark](https://cloud.githubusercontent.com/assets/2943616/11005191/81bc6604-84cd-11e5-8628-8a56c97d198f.png)
_Atom Dark_

<!-- -->
> ![one_dark](https://cloud.githubusercontent.com/assets/2943616/11005228/b62b1534-84cd-11e5-9096-264e76dcf7d9.png)
_One Dark_

<!-- -->
> ![monokai](https://cloud.githubusercontent.com/assets/2943616/11005242/c70231c6-84cd-11e5-91b4-0f7060e122b3.png)
_Monokai_

<!-- -->
> ![seti](https://cloud.githubusercontent.com/assets/2943616/11005246/cf1894c2-84cd-11e5-82c3-b017419f2fb9.png)
_Seti_

It seems `keyword.other` for `lambda`, `->` look most closer for existing Ruby syntax highlighting